.github/CODEOWNERS

```diff
@@ -369,8 +369,8 @@
 /comp/core/autodiscovery/listeners/cloudfoundry*.go  @DataDog/agent-integrations
 /comp/core/autodiscovery/listeners/snmp*.go   @DataDog/ndm-core
 /comp/core/autodiscovery/providers/           @DataDog/container-platform
-/comp/core/autodiscovery/providers/file*.go   @DataDog/agent-metrics-logs
-/comp/core/autodiscovery/providers/config_reader*.go @DataDog/container-platform @DataDog/agent-metrics-logs
+/comp/core/autodiscovery/providers/file*.go   @DataDog/agent-logs
```

Comment:
I'm not sure agent logs is the correct owner for this. This is regarding reading collecting agent config files from disk (not tailing logs)